### PR TITLE
Doesn't throw error when the application is not installed on the iOS simulator

### PR DESCRIPTION
**Pull request by Fatme**

### CODE REVIEW

File: lib/simctl.ts [hunk 1]
```diff
@@ -37,7 +37,16 @@ export class Simctl implements ISimctl {
 	}
 
 	public getAppContainer(deviceId: string, applicationIdentifier: string): IFuture<string> {
-		return this.simctlExec("get_app_container", [deviceId, applicationIdentifier]);
+		return (() => {
+			try {
+				return this.simctlExec("get_app_container", [deviceId, applicationIdentifier]).wait();
+			} catch(e) {
+				if (e.message.indexOf("No such file or directory") > -1) {
```
Comment (teobugslayer) [inline lib/simctl.ts]: I strongly recommend to explain in a comment why we have this code. It is very unnatural and two months late we won't remember why it is there.

That said, are we sure this is the *correct* way? Is it possible to get this message in other scenarios, thus trigger a false positive?

Comment (Fatme) [inline lib/simctl.ts]: We should get this message only in case when the application is not installed on the simulator.

Comment (e2l3n) [inline lib/simctl.ts]: Another approach would be to enumerate the directory contents of `~/Library/Developer/CoreSimulator/Devices/[DeviceID]/data/Containers/Data/Application/`. AFAIK we've depricated this approach for Xcode 7.0+ and in my opinion it is much cleaner and easier to use `simctl`.

**Comment by teobugslayer** [comment]
:+1: I've verified that the xcrun prints English messages on non-english OS. But again, document why the code is like that.

**Pull request merged**